routes.py

```diff
@@ -50,7 +50,7 @@ def DisplayClassPage(Class):
     
 @app.route('/DepartmentsMain/')
 def DepartmentsMainPage():
-    return render_template('DepartmentsMain.html')
+    return render_template('DepartmentsMain.html’)
```

Comment:
This here seems to be the source of the merge conflict. Go into my repo and copy what I have down (I believe lines 52-26 for the DepartmentsMainPage() function